include/nudb/context.hpp

```diff
@@ -77,6 +77,15 @@ class context
     void
     run();
 
+    /** Flush function.
+
+        Manually flushes all inserted objects to the disk.
+        If using this function, @ref start should not be called.
+    */
```

Comment:
The default basic_store constructor creates a context and starts it.

This comment should note that if the context object has been started manually or by the default constructor, a call to this function will be ignored and do nothing. The Flush function should return early if `num_threads_` isn't zero.

Reply:
Done.